docs-devsite/vertexai.md

```diff
@@ -267,9 +267,9 @@ export declare enum FunctionCallingMode
 
 |  Member | Value | Description |
 |  --- | --- | --- |
-|  ANY | <code>&quot;ANY&quot;</code> |  |
-|  AUTO | <code>&quot;AUTO&quot;</code> |  |
-|  NONE | <code>&quot;NONE&quot;</code> |  |
+|  ANY | <code>&quot;ANY&quot;</code> | Model is constrained to always predicting a function call only. If "allowed\_function\_names" is set, the predicted function call will be limited to any one of "allowed\_function\_names", else the predicted function call will be any one of the provided "function\_declarations". |
```

Comment:
Could these all be tagged as code?

- "allowed\_function\_names"
- "function\_declarations"

Reply:
yep, done!